<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLoginDialog</class>
 <widget class="QDialog" name="QLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>341</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="usernameLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="instructionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Please contact an administrator if you require an account or have forgotten your username or password.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QLoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
